provider/cmd/pulumi-resource-synced-folder/azure-blob-folder.ts

```diff
@@ -34,7 +35,7 @@ export class AzureBlobFolder extends pulumi.ComponentResource {
         args.managedObjects = args.managedObjects ?? true;
         args.disableManagedObjectAliases = args.disableManagedObjectAliases ?? false;
```

Comment:
I like being explicit about these defaults -- okay with this, to align with others?

```suggestion
        args.disableManagedObjectAliases = args.disableManagedObjectAliases ?? false;
        args.includeHiddenFiles = args.includeHiddenFiles ?? false;
```

Reply:
You shouldn't need this because the function parameter has a default value